Sources/Public/CalendarViewContent.swift

```diff
@@ -321,6 +352,7 @@ public final class CalendarViewContent {
 
   // TODO(BK): Make all item provider closures private(set) after legacy `CalendarItem` is removed.
   var monthHeaderItemModelProvider: (Month) -> InternalAnyCalendarItemModel
+  var monthFooterItemModelProvider: (Month) -> InternalAnyCalendarItemModel
```

Comment:
make this an optional closure and remove the default value

Reply:
Updated to optional and removed default value, now is set to nil